lib/lob/resources/resource_base.rb

```diff
@@ -12,7 +13,7 @@ def initialize(config)
         @config = config
       end
 
-      def list(query={})
+      def list(**query)
```

Comment:
This is actually doing two different things here:

`**query` is similar to node's `...query` in expressing variable arity. `query = {}` is assigning a default value to query if it's undefined. Not really sure what's changed here but I'll take a look locally to verify.

Reply:
This is actually a leftover from something I was experimenting with. I'll change it back.